h-youjin/LV.0/최댓값-만들기(1).js

```diff
@@ -0,0 +1,10 @@
+function solution(numbers) {
+  let answer = 0;
+
+  numbers.sort(function (a, b) {
+    return b - a;
+  });
+
+  answer = numbers[0] * numbers[1];
+  return answer;
+}
```

Comment:
```suggestion
function solution(numbers) {

  numbers.sort((a, b) =>  b - a);

  return numbers[0] * numbers[1];
}
```
<br />

- 잘푸셨네요!!!👏
- `sort()`메서드가 반환하는 문이 1개이기에 축약하여 쓰실 수 있습니다! 그리고 콜백함수는 특히 화살표 함수를 추천드립니다👍
- 꼭 answer에 할당하여 반환하실 필요는 없습니다!!! 이렇게 바로 반환해주셔도 좋습니다!!
- 그리고 `sort()` 메서드는 원본 배열 자체를 변경하는 메소드이기 때문에 아래 방법처럼도 가능합니다 참고만해주세요!!
```js
function solution(numbers) {
  return numbers.sort((a, b) => b - a)[0] * numbers[1];
}
```